### PR TITLE
Add price validity window param

### PR DESCRIPTION
**Pull request by leomassazza**

### CODE REVIEW

File: markets/perps-market/contracts/modules/AsyncOrderModule.sol [hunk 1]
```diff
@@ -22,7 +22,6 @@ contract AsyncOrderModule is IAsyncOrderModule {
     using DecimalMath for uint256;
     using DecimalMath for int64;
     using PerpsPrice for PerpsPrice.Data;
-    using AsyncOrder for AsyncOrder.Data;
```
Comment (leomassazza) [inline markets/perps-market/contracts/modules/AsyncOrderModule.sol:25]: remove duplicate

File: markets/perps-market/contracts/modules/PerpsAccountModule.sol [hunk 2]
```diff
@@ -21,7 +21,6 @@ contract PerpsAccountModule is IAccountModule {
     using AsyncOrder for AsyncOrder.Data;
     using SafeCastU256 for uint256;
     using SafeCastI256 for int256;
-    using PerpsAccount for PerpsAccount.Data;
```
Comment (leomassazza) [inline markets/perps-market/contracts/modules/PerpsAccountModule.sol:24]: remove duplicate

File: markets/perps-market/contracts/storage/AsyncOrder.sol [hunk 3]
```diff
@@ -7,7 +7,6 @@ import {IAsyncOrderModule} from "../interfaces/IAsyncOrderModule.sol";
 import {SettlementStrategy} from "./SettlementStrategy.sol";
 import {Position} from "./Position.sol";
 import {PerpsMarketConfiguration} from "./PerpsMarketConfiguration.sol";
-import {SettlementStrategy} from "./SettlementStrategy.sol";
```
Comment (leomassazza) [inline markets/perps-market/contracts/storage/AsyncOrder.sol:10]: duplicate import

File: markets/perps-market/contracts/storage/SettlementStrategy.sol [hunk 4]
```diff
@@ -26,6 +26,10 @@ library SettlementStrategy {
          * @dev the duration of the settlement window, after which committed orders can be cancelled.
          */
         uint256 settlementWindowDuration;
+        /**
+         * @dev the duration of the price window, after which price is not valid.
+         */
+        uint256 priceWindowDuration;
```
Comment (bachstatter) [inline markets/perps-market/contracts/storage/SettlementStrategy.sol:32]: We're not using setters for these? Is the idea that a new Strategy would be created if we want to update these?

**Review by bachstatter — APPROVED** [review]

File: markets/perps-market/contracts/storage/SettlementStrategy.sol (hunk 4 above)
Comment (leomassazza) [reply markets/perps-market/contracts/storage/SettlementStrategy.sol:32]: originally, yes. We can add later a module to update a settlement strategy if needed, but here and in spot, you can only create a new one and set the old as disabled.

**Review by sunnyvempati — COMMENTED** [review]
hold off on this until the settlement strategy refactor is in?

**Review by sunnyvempati — APPROVED** [review]
i think we are good to merge this.  we're probably keeping the current settlement strategy impl

**Pull request merged**